### DISCUSSION TITLE
Skin conductance level

### DISCUSSION THREAD

**Discussion by isabeln23** [neuropsychology/NeuroKit#704] [Q&A]
Hello, I wonder if neurokit provides a way to extract the mean skin conductance level for epochs in different events as a tonic measure, since the eda_intervalrelated function analyzes the SCR-related parameter. Thanks in advance!

**Comment by DominiqueMakowski** [comment]
Hi @isabeln23

After epoching the processed signals, you should have a dictionary of dataframes (one for each epoch) containing the different signals. You can then just create a loop over these epochs, extract then EDA_Tonic column from the dataframe, and average that signal to obtain what you want.
